.github/workflows/receive-fork-review.yml

```diff
@@ -0,0 +1,28 @@
+name: Receive Fork Review
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  approved:
+    # only run the workflow when PR is from a fork and when the PR is reviewed and approved
+    if: |
+      (github.event.pull_request.head.repo.full_name != github.repository) &&
+      (github.event.review.state == 'approved')
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "This PR is approved and e2e tests will begin running."
+  build:
+    needs: approved
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+      - name: Save commit ID
```

Comment:
Can you put a comment why we're using `upload-artifact`? Link to the GitHub security [post](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/) would help too.